server/prisma/generator/factories/chapters.factory.ts

```diff
@@ -12,7 +12,21 @@ const createChapters = async (userId: number): Promise<number[]> => {
     const name = company.companyName();
     const description = lorem.words();
     const category = lorem.word();
-
+    const tagNames = [
+      'Front End Development',
+      'Back End Development',
+      'FullStack Development',
+      'Cat Adoring',
+      'Dog Adoring',
+      'SNAKES',
+      'DevOps',
+      'Consulting',
+      'Hang Out',
+      'Security',
+    ];
+    const randomTag = Math.floor(Math.random() * tagNames.length);
+
+    const tag = tagNames[randomTag];
```

Comment:
You can use the random.ts library if you like. Doesn't really matter, but it exists if you're interested.

Reply:
> 2nd attempt to make it array

Oh, sorry, didn't see that!

Reply:
Nah, as always any tips and guidance are welcomed. I will check it out and note it when I display tags in the card